### PR TITLE
SPUI-911 > Add interest rate bottom sheet

### PR DESCRIPTION
**Pull request by VovaMedvediev**

### CODE REVIEW

File: ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -8,8 +8,8 @@
 
 /* Begin PBXBuildFile section */
 		1498D2341E8E89220040F4C2 /* GeneratedPluginRegistrant.m in Sources */ = {isa = PBXBuildFile; fileRef = 1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */; };
-		31B5F653267244BF00553BAF /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 31B5F652267244BF00553BAF /* GoogleService-Info.plist */; };
 		3B3967161E833CAA004F5970 /* AppFrameworkInfo.plist in Resources */ = {isa = PBXBuildFile; fileRef = 3B3967151E833CAA004F5970 /* AppFrameworkInfo.plist */; };
+		42BF60932785D73D0067C1F2 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 42BF60922785D73D0067C1F2 /* GoogleService-Info.plist */; };
```
Comment (eli1stark) [inline ios/Runner.xcodeproj/project.pbxproj:12]: is it possible to make commit without `project.pbxproj`

File: lib/app/shared/features/wallet/view/components/wallets_body/components/card_block/components/wallet_card.dart [hunk 2]
```diff
@@ -34,84 +42,218 @@ class WalletCard extends HookWidget {
...
-              value: marketItem.baseBalance,
-              accuracy: baseCurrency.accuracy,
-              symbol: baseCurrency.symbol,
+    return InkWell(
+      onTap: () {
+        sShowBasicModalBottomSheet(
+          context: context,
+          removeBarPadding: true,
+          horizontalPadding: 24,
+          children: [
```
Comment (eli1stark) [inline lib/app/shared/features/wallet/view/components/wallets_body/components/card_block/components/wallet_card.dart:51]: refactor please

File: lib/app/shared/features/wallet/view/components/wallets_body/components/card_block/components/wallet_card.dart [hunk 3]
```diff
@@ -34,84 +42,218 @@ class WalletCard extends HookWidget {
...
+                  '${currency.apy.toInt()}%',
+                  style: sBodyText1Style,
+                ),
+              ],
+            ),
+            const SpaceH40(),
+          ],
+        );
+      },
+      child: Container(
```
Comment (eli1stark) [inline lib/app/shared/features/wallet/view/components/wallets_body/components/card_block/components/wallet_card.dart:178]: and refactor this as well

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (VovaMedvediev) [reply ios/Runner.xcodeproj/project.pbxproj:12]: deleted changes in project.pbxproj

File: lib/app/shared/features/wallet/view/components/wallets_body/components/card_block/components/wallet_card.dart (hunk 2 above)
Comment (VovaMedvediev) [reply lib/app/shared/features/wallet/view/components/wallets_body/components/card_block/components/wallet_card.dart]: refactored

File: lib/app/shared/features/wallet/view/components/wallets_body/components/card_block/components/wallet_card.dart (hunk 3 above)
Comment (VovaMedvediev) [reply lib/app/shared/features/wallet/view/components/wallets_body/components/card_block/components/wallet_card.dart]: refactored

**Pull request merged**